worker/worker.go

```diff
@@ -42,16 +42,3 @@ type Infos interface {
 	GetDefault() (Worker, error)
 	WorkerInfos() []client.WorkerInfo
 }
-
-// Pre-defined label keys
-const (
-	labelPrefix              = "org.mobyproject.buildkit.worker."
```

Comment:
Did a quick search if there's any external consumers of these (besides vendoring), and doesn't seem to be the case , so probably no aliases and deprecation needed.

Reply:
Yes I https://grep.app/ and found nothing relevant